core/src/main/java/com/netflix/conductor/core/config/SchedulerConfiguration.java

```diff
@@ -15,23 +15,26 @@
 package com.netflix.conductor.core.config;
 
 import com.google.common.util.concurrent.ThreadFactoryBuilder;
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.scheduling.annotation.EnableAsync;
 import org.springframework.scheduling.annotation.EnableScheduling;
 import rx.Scheduler;
 import rx.schedulers.Schedulers;
 
-import java.util.concurrent.Executor;
-import java.util.concurrent.Executors;
-import java.util.concurrent.ThreadFactory;
-
 @Configuration(proxyBeanMethods = false)
 @EnableScheduling
 @EnableAsync
 public class SchedulerConfiguration {
```

Comment:
Since we have two `@Scheduled` annotation, we need to customize the executor used by Spring to have 2 threads, one for each `@Scheduled` method.

This class needs to implement `SchedulingConfigurer` and in the implemented method, we need to do something like,

```java
ThreadPoolTaskScheduler scheduler = new ThreadPoolTaskScheduler();
    scheduler.setThreadNamePrefix("scheduler-");
    scheduler.setPoolSize(2); // Equal to the number of jobs
    scheduler.initialize();
    taskRegistrar.setTaskScheduler(scheduler);
```

There may be a better way to do this.